routes/tripRouter.js

```diff
@@ -16,6 +16,22 @@ tripRouter.post('/', (req, res, next) => {
         })
 });
 
+tripRouter.get('/init/:id', (req, res, next) => {
+    const { id } = req.params;
+    // This route retrieves a trip's basic bag and todolist info
```

Comment:
Nitty nitpick: if you want to describe what routes do in comments, it might be easier to always place them in the same place: say, above the function

```js
// Retrieves a trip's basic bag and todolist info
tripRouter.get('/init/:id', (req, res, next) => {
```